posts/urls.py

```diff
@@ -1,8 +1,9 @@
 from django.urls import path
 
-from posts.views import PostListView, PostItemView
+from posts.views import PostListView, PostItemView, PostView
 
 urlpatterns = [
     path('/list', PostListView.as_view()),
     path('/<int:post_id>', PostItemView.as_view()),
+    path('/posting', PostView.as_view()),
```

Comment:
```suggestion
    path('', PostView.as_view()),
```
```python
POST :8000/posts
```
이렇게 POST 메소드를 사용하면 post를 생성하는 의미를 담기 때문에 posts/posting으로 작성하지 않으셔도 될 것 같아요!